README.md

```diff
@@ -1 +1,110 @@
+- [ ] Hannah uses a script to parallelize the setup of individual transformations in parallel using our cluster (`lilac`) via an [LSF bash script](https://github.com/choderalab/PersesAtFoldingAtHome/blob/master/SARS-CoV-2-Mpro/submit-ligpairs.sh) and a [python `run.py`](https://github.com/choderalab/PersesAtFoldingAtHome/blob/master/SARS-CoV-2-Mpro/run.py).
+- [ ] To set up a single ligand transformation (`i -> j`), we run the `perses` command-line entry point `perses-fah config.yaml`, where `config.yaml` looks like this:
+```yaml
+protein_pdb: CDK2_protein.pdb
+ligand_file: CDK2_ligands_shifted.sdf
+old_ligand_index: 0
+new_ligand_index: 1
+forcefield_files:
+    - amber/ff14SB.xml # ff14SB protein force field
+    - amber/tip3p_standard.xml # TIP3P and recommended monovalent ion parameters
+    - amber/tip3p_HFE_multivalent.xml # for divalent ions
+    - amber/phosaa10.xml # HANDLES THE TPO
+atom_expression:
+    - IntType
+bond_expression:
+    - DefaultBonds
+complex_projid: 13418
+solvent_projid: 13419
+```
+In practice, this is generated from a JSON file with a script called `run.py`.
+This requires a GPU to be practical (e.g. GTX 1080).
+
+This will create files `{complex_projid}/RUNS/RUN/...` and `{solvent_projid}/RUNS/RUN/...` with the directories populated with some metadata important to retain plus `{system,state,integrator,core}.xml` (or `.bz2` or `.gz` compressed versions thereof).
+- [ ] `rsync` these files over to `server@aws3.foldingathome.org:/projects/available/covid-moonshot/`. It's OK to add more RUNs dynamically, but they must be compact and sequentially numbered from `RUN0`. The `{projid}` directory will be symlinked to `/home/server/server2/projects/{projid}` for the server to discover it.
+- [ ] The `{projid}` directory will contain a `project.xml` file that looks like this:
+```XML
+<project type="OPENMM_22" id="13418">
+  <gpu/>
+  <min-core-version v="0.0.11"/>
+
+  <job-priority v="queue-count gen clone run"/>
+
+  <runs v="759"/>
+  <clones v="100"/>
+  <gens v="6"/>
+  <atoms v="89709"/>
+
+  <timeout v="1.0"/>
+  <deadline v="2.0"/>
+  <!-- <k-factor v="0.75"/> -->
+  <!-- <stats-credit v="53245"/> -->  
+  <!-- <stats-credit v="70000"/> --> <!-- SARS-CoV-2 Mpro -->
+  <stats-credit v="55000"/>
+
+  <send>
+    $home/core.xml
+    $home/RUNS/RUN$run/system.xml.bz2
+    $home/RUNS/RUN$run/integrator.xml
+    state.xml.bz2
+  </send>
+
+  <create-command>
+    ln -s -f $home/RUNS/RUN$run/state.xml.bz2 $jobdir/state.xml.bz2
+  </create-command>
+
+  <next-gen-command>
+    mv -f $results/checkpointState.xml.bz2 $jobdir/state.xml.bz2
+  </next-gen-command>
+
+  <!-- Don't copy to S3 storage gateway -->
+  <archive-command>
+    /usr/bin/true
```

Comment:
Would anything prevent us from archiving to S3 at this point, and running further analysis in the cloud (e.g. in a Lambda triggered by the presence of new data?)